packages/react/src/Autocomplete/AutocompleteInput.tsx

```diff
@@ -52,14 +52,14 @@ const AutocompleteInput = React.forwardRef(
     const [highlightRemainingText, setHighlightRemainingText] = useState<boolean>(true)
     const {safeSetTimeout} = useSafeTimeout()
 
-    const handleInputFocus: FocusEventHandler<HTMLInputElement> = useCallback(
+    const handleInputClick: MouseEventHandler<HTMLInputElement> = useCallback(
```

Comment:
Feel free to ignore this since it's not really related to the changes proposed 👀 

I wanted to ask if this `useCallback` is doing much for us or if it would be something we could refactor to a default event handler / function.

Reply:
Yeah we definitely can adjust this!